app/assets/javascripts/accessible-autocomplete.min.js

```diff
@@ -0,0 +1,2 @@
,t[0],t[1],t[2],t[3])}return e.apply(n,t)}},function(e,t,n){n(59)("match",1,function(o,r,e){return[function(e){"use strict";var t=o(this),n=e==undefined?undefined:e[r];return n!==undefined?n.call(e,t):new RegExp(e)[r](String(t))},e]})},function(e,t,n){"use strict";var l=n(7),s=n(16),a=n(4),c=n(12),p=n(34);e.exports=function(t,e,n){var o=p(t),r=n(c,o,""[t]),i=r[0],u=r[1];a(function(){var e={};return e[o]=function(){return 7},7!=""[t](e)})&&(s(String.prototype,t,i),l(RegExp.prototype,o,2==e?function(e,t){return u.call(e,this,t)}:function(e){return u.call(e,this)}))}},function(e,t,n){"use strict";t.__esModule=!0,t["default"]=void 0,n(37);var b=n(5);var o=function o(r,i,u){var l;return function(){var e=this,t=arguments,n=function n(){l=null,u||r.apply(e,t)},o=u&&!l;clearTimeout(l),l=setTimeout(n,i),o&&r.apply(e,t)}},r=function(r){function e(){for(var e,t=arguments.length,n=new Array(t),o=0;o<t;o++)n[o]=arguments[o];return(e=r.call.apply(r,[this].concat(n))||this).state={bump:!1,debounced:!1},e}(function n(e,t){e.prototype=Object.create(t.prototype),(e.prototype.constructor=e).__proto__=t})(e,r);var t=e.prototype;return t.componentWillMount=function(){var e=this;this.debounceStatusUpdate=o(function(){if(!e.state.debounced){var t=!e.props.isInFocus||e.props.validChoiceMade;e.setState(function(e){return{bump:!e.bump,debounced:!0,silenced:t}})}},1400)},t.componentWillReceiveProps=function(e){e.queryLength;this.setState({debounced:!1})},t.render=function(){var e=this.props,t=e.id,n=e.length,o=e.queryLength,r=e.minQueryLength,i=e.selectedOption,u=e.selectedOptionIndex,l=e.tQueryTooShort,s=e.tNoResults,a=e.tSelectedOption,c=e.tResults,p=this.state,f=p.bump,d=p.debounced,h=p.silenced,m=o<r,v=0===n,y=i?a(i,n,u):"",_=null;return _=m?l(r):v?s():c(n,y),this.debounceStatusUpdate(),(0,b.createElement)("div",{style:{border:"0",clip:"rect(0 0 0 0)",height:"1px",marginBottom:"-1px",marginRight:"-1px",overflow:"hidden",padding:"0",position:"absolute",whiteSpace:"nowrap",width:"1px"}},(0,b.createElement)("div",{id:t+"__status--A",role:"status","aria-atomic":"true","aria-live":"polite"},!h&&d&&f?_:""),(0,b.createElement)("div",{id:t+"__status--B",role:"status","aria-atomic":"true","aria-live":"polite"},h||!d||f?"":_))},e}(b.Component);(t["default"]=r).defaultProps={tQueryTooShort:function(e){return"Type in "+e+" or more characters for results"},tNoResults:function(){return"No search results"},tSelectedOption:function(e,t,n){return e+" "+(n+1)+" of "+t+" is highlighted"},tResults:function(e,t){return e+" "+(1===e?"result":"results")+" "+(1===e?"is":"are")+" available. "+t}}},function(e,t,n){"use strict";t.__esModule=!0,t["default"]=void 0;var o=n(5),r=function i(e){var t=e.className;return(0,o.createElement)("svg",{version:"1.1",xmlns:"http://www.w3.org/2000/svg",className:t,focusable:"false"},(0,o.createElement)("g",{stroke:"none",fill:"none","fill-rule":"evenodd"},(0,o.createElement)("polygon",{fill:"#000000",points:"0 0 22 0 11 17"})))};t["default"]=r}])["default"]});
```

Comment:
Instead of including the minified JavaScript and CSS (which is mostly unreadable), would we be able to include the component in the same way it has been done in Whitehall?  Then we won't need to try and keep these up-to-date ourselves and can use dependabot to that for us.

E.g. https://github.com/alphagov/whitehall/blob/3fdc3561228862058b831527c3bab22ae24a73ff/yarn.lock#L180-L184

Reply:
I should have said in the PR description, I tried this https://github.com/alphagov/transition/pull/1468

However, I'm struggling to install packages from repos with Yarn v3 (Transition uses Yarn 3, Whitehall uses Yarn 1), so I opted for the easiest solution which was this https://github.com/alphagov/accessible-autocomplete?tab=readme-ov-file#as-a-plain-javascript-module

You can see the error by running `yarn set version 3.4.1 && yarn add accessible-autocomplete-multiselect@alphagov/accessible-autocomplete-multiselect` vs `yarn set version 1.22.19 && yarn add accessible-autocomplete-multiselect@alphagov/accessible-autocomplete-multiselect`

Maybe I'll raise this in dev chat!